types/three/three-core.d.ts

```diff
@@ -6065,6 +6051,26 @@ export class WebGLLights {
     get(light: any): any;
 }
 
+/** 
+ * An object with a series of statistical information about the graphics board memory and the rendering process. 
+ */ 
+export class WebGLInfo { 
+    autoReset: boolean; 
+    memory: { 
+        geometries: number; 
+        textures: number; 
+    }; 
+    programs: number | null; 
```

Comment:
I don't think this is the right type for `programs` it appears to be set here: https://github.com/mrdoob/three.js/blob/bd63a6cc8dd7fe11b530410fab9f67a9de359be8/src/renderers/WebGLRenderer.js#L276